src/.schema.sql

```diff
@@ -1,13 +1,13 @@
 CREATE SCHEMA kubercade AUTHORIZATION nrhwqomw
```

Comment:
I think we should make this
`CREATE SCHEMA IF NOT EXISTS kubercade AUTHORIZATION nrhwqomw`
Then you can run the whole schema at any time without errors.